### ISSUE TITLE
Properly integrate CLA agreement into the workflow

### ISSUE THREAD

**Issue by Floppy** [openpolitics/votebot#72]
Currently we just link to CLAhub, which is a bit lame. We could include it in the workflow directly, either using their API (which might involve improving it), or just having our own CLA which is agreed to as part of the flow.

**Comment by jt-nti** [comment]
It would be nice to remove the dependency on CLAhub. Feedback on the Eastleigh manifesto deployment of groupthink so far is predominantly on how confusing it is to get started and eliminating the CLAhub authentication step would help simplify things.

**Comment by Floppy** [comment]
Yeah, it would be much better to just include CLA text in the "submit PR" page.
